patches/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -132,11 +132,50 @@
              if (this.has(DataComponents.CUSTOM_NAME)) {
                  mutablecomponent.withStyle(ChatFormatting.ITALIC);
              }
-@@ -761,6 +_,7 @@
+ 
+             list.add(mutablecomponent);
++            net.neoforged.neoforge.common.NeoForge.EVENT_BUS.post(new net.neoforged.neoforge.event.entity.player.ItemTooltipEvent.AfterName(this, p_41652_, list, p_41653_, p_339637_));
```

Comment:
Suggestion: move this to ClientHooks and introduce an enum for ItemTooltipPosition.